src/components/Header.jsx

```diff
@@ -0,0 +1,15 @@
+import React from 'react';
+import style from '../style/Header.module.css';
+import { useNavigate } from 'react-router-dom';
+
+export default function Header() {
+  const navigate = useNavigate();
+  return (
+    <div className={style.container}>
+      <button onClick={() => navigate(-1)} className={style.back}>
+        Back
+      </button>
```

Comment:
- [ ] [REQUIRED] Kindly ensure that the `Back` button is not been displayed on the `Home` page for a better UI.